### PR TITLE
params: Add query parameters for last login and last discharge times

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Comment by jujugui** [comment]
Build finished.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/idmclient/96/

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/idmclient-merge

File: params/params.go [hunk 1]
```diff
@@ -84,8 +84,24 @@ type LoginMethods struct {
...
 type QueryUsersRequest struct {
 	httprequest.Route `httprequest:"GET /v1/u"`
-	ExternalID        string `httprequest:"external_id,form"`
-	Email             string `httprequest:"email,form"`
+
+	// ExternalID, if present, matches all identities with the given
+	// external ID (there should be a maximum of 1).
+	ExternalID string `httprequest:"external_id,form"`
+
+	// EMail, if present, matches all identities with the given email
```
Comment (rogpeppe) [inline params/params.go]: s/EMail/Email/

**Pull request merged**